(dp1
S'output'
p2
S"<type 'exceptions.NameError'> global name 'form' is not defined"
p3
sS'layer'
p4
S'/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7fdbc80f0cb0, file "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/default.py", line 7>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/default.py", line 159, in <module>\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/gluon/globals.py", line 421, in <lambda>\n    self._caller = lambda f: f()\n  File "/mnt/c/Users/Me/Desktop/UCSC/CS183/said-it/applications/init/controllers/default.py", line 15, in add\n    return dict(form=form)\nNameError: global name \'form\' is not defined\n'
p11
s.